superset-frontend/plugins/legacy-preset-chart-nvd3/src/NVD3Vis.js

```diff
@@ -844,12 +844,14 @@ function nvd3Vis(element, props) {
       }
 
       chart.yDomain1([
-        yAxisBounds[0] || ticks1[0],
-        yAxisBounds[1] || ticks1[ticks1.length - 1],
+        isDefined(yAxisBounds[0]) ? yAxisBounds[0] : ticks1[0],
```

Comment:
nit: how about using [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)?

```suggestion
      yAxisBounds[0] ?? ticks1[0],
```